Zukini.API/Steps/ApiSteps.cs

```diff
@@ -1,167 +1,37 @@
 ﻿using BoDi;
-using RestSharp;
-using System;
-using System.Collections.Generic;
-using System.Net;
 using Zukini.Steps;
 
 namespace Zukini.API.Steps
 {
-    public abstract class ApiSteps : BaseSteps
+    public abstract class ApiSteps : ApiSteps<StepsContext>
```

Comment:
We have 2 files with a name ApiSteps?

Reply:
ApiSteps<> is the generic
ApiSteps provided backwards compatibility as it's just a wrapper for ApiSteps<StepsContext>

Previously, ApiSteps and UISteps inherited from BaseSteps forming a Is-A relationship.  This gave them the property bucket and test id.  I found a need for something in the SpecFlow Step implementation and neither Zukini Step inherits from the SpecFlow Step.

The Is-A relationship just to avoid code duplication isn't very helpful so introduced the generic so that Zukini Steps "Have A" property bucket and test id (contained in a context).  Now, users can specify their own context that may have their own custom property bucket or other meta data.  I can also inherit Zukini Steps from SpecFlow Steps.